roles/openshift_efk/tasks/main.yaml

```diff
@@ -0,0 +1,179 @@
+---
+
+  - fail: msg="This role requires the following vars to be defined: master_url, kibana_hostname, es_cluster_size"
+    when: "kibana_hostname is not defined or
+          es_cluster_size is not defined or
+          master_url is not defined"
+
+  - name: "Checking for logging project"
+    command: oc get project logging
+    register: logging_project
+    failed_when: "'FAILED' in logging_project.stderr"
+    tags:
+      - cleanup
+
+  - name: "Create logging project"
+    command: oadm new-project logging
+    when: logging_project.rc != 0
+    tags:
+      - build
+  - name: "Changing projects"
+    command: oc project logging
```

Comment:
Since this mutates the default user config, I would recommend that instead you should copy the admin kubeconfig to a temporary location and then use that configuration file for the oc/oadm commands instead.

The openshift_persistent_volumes role has a good example of this. temp file creation: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_persistent_volumes/tasks/main.yml#L2-L10 temp file cleanup: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_persistent_volumes/tasks/main.yml#L46-L50